src/pages/patientView/PatientViewPage.tsx

```diff
@@ -22,7 +22,8 @@ import {getSpans} from './clinicalInformation/lib/clinicalAttributesUtil.js';
 import CopyNumberTableWrapper from "./copyNumberAlterations/CopyNumberTableWrapper";
 import {reaction, computed} from "mobx";
 import Timeline from "./timeline/Timeline";
-import {default as PatientViewMutationTable, MutationTableColumnType} from "./mutation/PatientViewMutationTable";
+import {default as PatientViewMutationTable} from "./mutation/PatientViewMutationTable";
+import {MutationTableColumnType} from "shared/components/mutationTable/MutationTable"
 import PathologyReport from "./pathologyReport/PathologyReport";
 import { MSKTabs, MSKTab } from "../../shared/components/MSKTabs/MSKTabs";
 import validateParameters from '../../shared/lib/validateParameters';
```

Comment:
I was thinking that we'd move `mutationTableColumns` on line 90, move all that stuff into PatientViewMutationTable